### PR TITLE
hotfix: Dummy PR to check the changes over v1.46 (DO NOT MERGE) 

### PR DESCRIPTION
**Pull request by abhvsn**
## Description
Dummy PR to check the diffs between v1.46 and hotfix tag will be created from branch `hotfix/v1.46.1`

### CODE REVIEW

File: deploy/docker/fs/opt/appsmith/entrypoint.sh [hunk 1]
```diff
@@ -467,8 +467,15 @@ create_appsmith_pg_db() {
   # Start the postgres , wait for it to be ready and create a appsmith db
   su postgres -c "env PATH='$PATH' pg_ctl -D $POSTGRES_DB_PATH -l $POSTGRES_DB_PATH/logfile start"
   echo "Waiting for Postgres to start"
+  local max_attempts=100
```
Comment (trishaanand) [inline deploy/docker/fs/opt/appsmith/entrypoint.sh:470]: What happens when postgres fails to start even after 100 seconds?

**Review by trishaanand — APPROVED** [review]

File: deploy/docker/fs/opt/appsmith/entrypoint.sh (hunk 1 above)
Comment (abhvsn) [reply deploy/docker/fs/opt/appsmith/entrypoint.sh:470]: We are disabling the emebdded postgres completely, and supervisor will not restart the embedded postgres.

Comment (trishaanand) [reply deploy/docker/fs/opt/appsmith/entrypoint.sh:470]: Thanks! That makes sense.

**Comment by abhvsn** [comment]
Closing this as this was a dummy PR and hotfix v1.46.1 is released.

**Pull request closed without merging**